checkbox-support/checkbox_support/dbus/gnome_monitor.py

```diff
@@ -104,6 +106,107 @@ def set_extended_mode(self) -> Dict[str, str]:
         self._apply_monitors_config(state[0], extended_logical_monitors)
         return configuration
 
+    def cycler(
+        self,
+        res: bool = True,
+        max_res_amount: int = 5,
+        trans: bool = False,
+        log: Callable[..., Any] = None,
+        action: Callable[..., Any] = None,
+        **kwargs
+    ):
+        """
+        Cycling change the monitors configurations automatically
+
+        Args:
+            res: Cycling the resolution or not.
+
+            max_res_amount: The supported resolution might be a lot
+                            and this could limited test amount.
```

Comment:
For clearness
```suggestion
            max_res_amount: Limit the number of tested configurations
                to avoid an unnecessarily large test matrix.
```